src/orchestrator-tracing/src/lib.rs

```diff
@@ -179,6 +179,17 @@ pub struct TracingCliArgs {
     /// See: <https://docs.sentry.io/platforms/rust/configuration/options/#environment>
     #[clap(long, env = "SENTRY_ENVIRONMENT")]
     pub sentry_environment: Option<String>,
+    /// Additional tags to send with all sentry events. These are added to
+    /// `--opentelemetry-resource`
```

Comment:
(The current phrasing makes it sound like we attach the Sentry tags to OpenTelemetry traces, which we do not!)

```suggestion
    /// Tags to send with all Sentry events, in addition to the tags
    /// specified by `--opentelemetry-resource`.
```